### DISCUSSION TITLE
FileUpload with editor creating two images in tmp folder

### DISCUSSION THREAD

**Discussion by jason-beach** [filamentphp/filament#8070] [Help]
Anyone have issues with the FileUpload with the image editor with creating two images in the tmp directory? 

So what's happening is when I'm creating a new record (create form) in a repeater, I upload a file and I see it's put in the livewire-tmp directory - all good there. However, when I edit that image prior to creating the record, the cropped image is also stored in the livewire-tmp directory. Then when I create the record or save, both versions of the image get copied over to the public directory (for a brief moment you can see both files in filepond - watch video) and when I edit my record the preview of my image is the original image not the cropped version and the original image name is stored in the database.

Shouldn't the original image get overwritten when using the image editor's cropped version prior to saving a record? There should only be one image being copied from the livewire-tmp to the public not two. 

Here's a video of my experience:
https://app.screencast.com/SuC8HUNZa25J6

Anyone else experiencing this or is this a bug or am I missing something?

**Comment by wdandrewjames** [comment]
I'm having the same issue. Trying to find a way around this.

**Comment by wychoong** [comment]
create a simple repo and submit issue
